src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/CustomObject.txt

```diff
@@ -11,4 +11,13 @@
 "s"
 
 >> Sum(Table(ParseJson("[1, 2, 3, 4, 5]")), Value(Value))
-15
\ No newline at end of file
+15
+
+>> Value(ParseJson("{""a"": 5}").a)
+5
+
+>> Value(ParseJson("{""a"": 5}").b)
+Blank()
+
+>> Value(ParseJson("""s""").a)
+#Error
```

Comment:
What about something like 
`Value(ParseJson("This "" Is , "" Invalid ").a`?

Reply:
This should be a compile error because the result of value is a number and cannot be used with the dot operator

Reply:
ah, whoops, meant 
Value(ParseJson("This "" Is , "" Invalid ").a)